apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -21,7 +21,8 @@
               "
               :clone="clone"
               :locale="locale"
-              @errorsInForm="handleErrors"
+              :errorPerColumn="rowErrors"
+              @update:model-value="checkForErrors"
```

Comment:
v-modal is a shorthand for in and output. By handeling the output yourself and having a v-model you will now have 2 channels outgoing on 'model-value'.  I suggest using ':modal-value' for input if you already defined a custom output.

Reply:
redacted :)

Reply:
This is on purpose, since I wanted both behaviours. Seemed better to not explicitly write the normal v-model behaviour myself.